RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -140,7 +140,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
 
   namespace pt3dnn {
 
-    template <typename TAcc>
+    template <bool IsPT5, typename TAcc>
```

Comment:
```suggestion
    template <bool isPT5WP, typename TAcc>
```
perhaps would be more clear that it's just the cut.

Eventually it may be better to break up the inference and the cut application part and template only that.